### ISSUE TITLE
But report dialog improvement

### ISSUE THREAD

**Issue by GoogleCodeExporter** [Risehat/freemedforms#290]
```
When user make a bug report , always ask him if he did check thé help of the 
app or website
```

Original issue reported on code.google.com by `eric.mae...@gmail.com` on 7 Feb 2013 at 6:57

**Comment by GoogleCodeExporter** [comment]
```
I'll do that. Good idea.
```

Original comment by `christian.a.reiter@gmail.com` on 7 Feb 2013 at 10:10
- Changed state: **Started**

**Comment by GoogleCodeExporter** [comment]
```
This issue was closed by revision e2c3314cc310.
```

Original comment by `eric.mae...@gmail.com` on 22 Feb 2013 at 6:00
- Changed state: **Fixed**
